### PR TITLE
#4 - UI Design

### PR DESCRIPTION
**Pull request by limkokyew**
Some aspects of the UI could still be improved, but it's more important right now to get the functionality of the app working.

### CODE REVIEW

**Review by mac641 — APPROVED** [review]
👍🏿 I guess

**Comment by mac641** [comment]
There seems to be an unused import in ImageAnalyzer though...

**Review by mac641 — APPROVED** [review]

File: app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire03Fragment.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            inflater,
+            R.layout.fragment_questionnaire_03,
+            container,
+            false
+        )
+
+        binding.seekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
+            override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
+                binding.seekBarProgressLabel.text = seekBar.progress.toString()
+                viewModel.questionnaire01Answer = progress
```
Comment (AnnaGsell) [inline app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire03Fragment.kt:36]: questionnaire01Answer is wrong, we're in 03

File: app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire05Fragment.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            inflater,
+            R.layout.fragment_questionnaire_05,
+            container,
+            false
+        )
+
+        binding.seekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
+            override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
+                binding.seekBarProgressLabel.text = seekBar.progress.toString()
+                viewModel.questionnaire01Answer = progress
```
Comment (AnnaGsell) [inline app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire05Fragment.kt:36]: See above

File: app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire06Fragment.kt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            inflater,
+            R.layout.fragment_questionnaire_06,
+            container,
+            false
+        )
+
+        binding.seekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
+            override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
+                binding.seekBarProgressLabel.text = seekBar.progress.toString()
+                viewModel.questionnaire01Answer = progress
```
Comment (AnnaGsell) [inline app/src/main/java/de/hsaugsburg/teamulster/sohappy/fragment/Questionnaire06Fragment.kt:36]: See above

**Review by craftamap — APPROVED** [review]

**Pull request merged**